Add page about session (#480)
shopware/docs: guides/hosting/performance/session.md

# Shopware Session

> Shopware uses the default PHP configured storage for session. This is in most systems' file system. So in smaller setups you will not need to take care of the sessions. 

Reviewer: ```suggestion
Shopware, by default, uses the session storaged configured in PHP. On most installations, this is the file system. So, in smaller setups, you will not need to take care of sessions. 
```

> For larger setups with much traffic or clustering, however, you will most probably want to configure alternative session storage such as Redis in order to reduce the load on the database.

Reviewer: ```suggestion
For larger setups with a lot of traffic, or that are using clustering however, you will most probably want to configure alternative session storage such as Redis, in order to reduce the load on the database.
```

## Session Adapters

### Configure Redis using PHP.ini

> As Shopware uses by default the PHP configured settings, you can configure the Session config directly in your `php.ini`. Here is an example for configure it directly in PHP.

Reviewer: ```suggestion
As Shopware by default uses settings configured in PHP, you can reconfigure the Session config directly in your `php.ini`. Here is an example to configure it directly in PHP.
```

```ini
session.save_handler = redis
session.save_path = "tcp://host:6379"
```

> See official [PhpRedis documentation](https://github.com/phpredis/phpredis#php-session-handler) for all possible options

Reviewer 2: ```suggestion
Please refer to the official [PhpRedis documentation](https://github.com/phpredis/phpredis#php-session-handler) for all possible options.
```

### Configure Redis using Shopware configuration

> If you don't have access to the php.ini, you can configure it directly in Shopware itself. For this create a `config/packages/redis.yml` file with following content

Reviewer 2: ```suggestion
If you don't have access to the php.ini configuration, you can configure it directly in Shopware itself. For this create a `config/packages/redis.yml` file with the following content
```

```yaml
# config/packages/redis.yml
framework:
    session:
        handler_id: "redis://host:port"
```

### Other adapters

> Symfony provides also PHP implementations of some adapters:

Reviewer 2: ```suggestion
Symfony also provides PHP implementations of some adapters:
```
